### ISSUE TITLE
We need an "AWS" version of this

### ISSUE THREAD

**Issue by Coinhexa** [clemlesne/scrape-it-now#8]
That uses all the AWS components to realize the same application

**Comment by clemlesne** [comment]
Good idea if that's fit your practice!

In than case, we could:

- Add abstractions for
  - Queue (Queue Storage / SQS)
  - Blob (Blob Storage / S3)
  - Monitoring (App Insights / Distro for OpenTelemetry)
- Add a configuration file with a switch + config from those providers

**Comment by clemlesne** [comment]
First part of the job is done [with the v1.5.0](https://github.com/clemlesne/scrape-it-now/releases/tag/v1.5.0): Local Storage has been added in addition to Azure Storage.

You can now implement the AWS SDKs and push your PR!

Also, in the meantime, [Mountpoint for Amazon S3](https://aws.amazon.com/s3/features/mountpoint/) can be a viable option for you.

**Comment by clemlesne** [comment]
Hello @Coinhexa, do you still need that?

**Comment by Coinhexa** [comment]
> Hello @Coinhexa, do you still need that?

Yep very much needed. AWS is where 70% of the cloud market it, this library would see a bigger influx of users with it

**Comment by clemlesne** [comment]
Work started on `feat/aws-peristence`.

**Comment by clemlesne** [comment]
@Coinhexa can you review the PR https://github.com/clemlesne/scrape-it-now/pull/43?
